index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <div id="main">
```

Comment:
- [ ] Your explanation here looks amazing however, it's easily confusing for the student to understand which piece of the code you are requesting changes, Kindly be more specific 💪🏼 

**eg:**

Using semantic tags such as `section` instead of `div` will help increase.....for that reason please consider changing the `div` tag to `semantic` tag 👏🏼